src/tasks/DockerBuildTaskDefinitionBase.ts

```diff
@@ -3,14 +3,15 @@
  *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import { ContainerPlatform } from '../runtimes/docker';
 import { DockerLabels, TaskDefinitionBase } from './TaskDefinitionBase';
 
 export interface DockerBuildOptions {
     buildArgs?: { [key: string]: string };
     context?: string;
     dockerfile?: string;
     labels?: DockerLabels;
-    platform?: string;
+    platform?: ContainerPlatform;
```

Comment:
The ContainerPlatform type should be declared as part of TaskDefinitionBase or in this file and not come from the runtime types.

Reply:
We want to make sure we don't have our client options and task options too tightly coupled.